napari/utils/events/evented_model.py

```diff
@@ -1,16 +1,54 @@
 import operator
+import sys
 import warnings
+from contextlib import contextmanager
 from typing import Any, Callable, ClassVar, Dict, Set
 
-from pydantic import BaseModel, PrivateAttr
-from pydantic.main import ModelMetaclass
+from pydantic import BaseModel, PrivateAttr, main, utils
 
 from ...utils.misc import pick_equality_operator
 from .custom_types import JSON_ENCODERS
 from .event import EmitterGroup, Event
 
 
-class EqualityMetaclass(ModelMetaclass):
+@contextmanager
+def no_class_attributes():
+    """Context in which pydantic.main.ClassAttribute just passes value 2.
+
+    Due to a very annoying decision by PySide2, all class ``__signature__``
```

Comment:
Can you link to the issue where you raised #2264, maybe even this PR #2265 too, I can just imagine someone who reads this wanting to find there way back here too